docs/diagrams/plantuml/Find.puml

```diff
@@ -0,0 +1,14 @@
+@startuml
+'https://plantuml.com/activity-diagram-beta
+
+start
+:User executes find command;
+if (is Field given) then (true)
```

Comment:
I think there is an error in the branch node notation.

Reply:
Thanks for pointing it out, I'll rectify it!